egs2/wsj0_2mix/enh1/run.sh

```diff
@@ -21,5 +21,5 @@ test_sets="tt_${min_or_max}_${sample_rate} "
     --lang en \
     --ngpu 1 \
     --local_data_opts "--sample_rate ${sample_rate} --min_or_max ${min_or_max}" \
-    --enh_config ./conf/tuning/train_enh_PSM.yaml \
+    --enh_config ./conf/tuning/train_enh_rnn.yaml \
```

Comment:
Seems the train_enh_rnn.yaml changed to train_enh_rnn_tf.yaml.
Please check it in new PR. @LiChenda